packages/text-split/index.tsx

```diff
@@ -32,7 +52,7 @@ function TextSplit({
             </h2>
           )}
           {hasStringContent ? (
-            <ContentString contentString={content} theme={theme} />
+            <ContentString contentString={content} />
```

Comment:
Why'd we remove the theme prop here?

Reply:
Because `ContentString` only uses the `contentString` prop. https://github.com/hashicorp/react-components/blob/main/packages/text-split/index.js#L53